docs/manifest.json

```
@@ -860,15 +860,6 @@
         { "title": "Emails" },
         [["createEmail()", "/references/backend/emails/create-email"]]
       ],
```

Gives an error for trailing comma (causing build fail) 

```suggestion
      ]
```